definitions/features/puma.rb

```diff
@@ -0,0 +1,18 @@
+module ForemanMaintain
+  module Features
+    class Puma < ForemanMaintain::Feature
```

Comment:
Is it possible to name this something related to foreman-service given thats the systemd service?

Reply:
I agree here too, we don't need to call out puma.

Reply:
Renamed feature `puma` to `foreman_service`.